plugin/al/lib/AL_USDMaya/AL/usdmaya/nodes/TransformationMatrix.h

```diff
@@ -294,6 +294,78 @@ class TransformationMatrix
   /// return  true if written ok
   static bool pushMatrix(const MMatrix& input, UsdGeomXformOp& op, UsdTimeCode timeCode = UsdTimeCode::Default());
 
+  /// \brief  If set to true, transform values will target the animated key-frame values in the prim. If set to false,
+  ///         the transform values will target the default attribute values.
+  /// \param  enabled true to target animated values, false to target the default.
+  void enableReadAnimatedValues(bool enabled);
+
+  /// \brief  Applies a local space translation offset to the computed matrix. Useful for positioning objects on a
+  ///         table.
+  /// \param  localTranslateOffset the local space offset to apply to the transform.
+  inline void setLocalTranslationOffset(const MVector& localTranslateOffset)
+    { m_localTranslateOffset = localTranslateOffset; }
+
+  /// \brief  return the prim this transform matrix is attached to
+  /// \return the prim this transform matrix is controlling
+  inline const UsdPrim& prim() const
```

Comment:
/git/usd/maya-usd/plugin/al/lib/AL_USDMaya/AL/usdmaya/nodes/TransformationMatrix.h:310:25: error: 'prim' overrides a member function but is not marked 'override' [-Werror,-Winconsistent-missing-override]
  inline const UsdPrim& prim() const
                        ^
/git/usd/maya-usd/plugin/al/lib/AL_USDMaya/AL/usdmaya/nodes/BasicTransformationMatrix.h:73:26: note: overridden virtual function is here
  virtual const UsdPrim& prim() const

Reply:
In addition, there is no value in making the virtual method inlined.